x-pack/docs/en/security/authentication/configuring-pki-realm.asciidoc

```diff
@@ -41,15 +33,21 @@ xpack:
 With this configuration, any certificate trusted by the {es} SSL/TLS layer is
 accepted for authentication. The username is the common name (CN) extracted
 from the DN in the Subject field of the end-entity certificate. This
-configuration does not permit PKI authentication to {kib}.
+configuration is not sufficient to permit PKI authentication to {kib};
```

Comment:
> is not sufficient

I changed this sentence since by my understanding this step is *necessary* to permit PKI authentication in Kibana, it just isn't sufficient (other steps are required).  If I've misunderstood, please let me know and I can revert this change.

Reply:
This is correct!